packages/next/src/data/server/handleError.ts

```diff
@@ -96,11 +96,13 @@ export async function handleError(
 						destination: redirect.location,
 						permanent: redirect.status === 301 || redirect.status === 308,
 					},
-				};
+				} satisfies GetServerSidePropsResult<Record<string, never>>;
```

Comment:
Any particular reason for using `never` over `unknown` here?

Reply:
don't think so, `unknown` should do the trick here. I'll update